diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -412,8 +441,24 @@ controller_interface::CallbackReturn DiffDriveController::on_configure(
           "time, this message will only be shown once");
         msg->header.stamp = get_node()->get_clock()->now();
```

Comment:
```suggestion
        msg->header.stamp = get_node()->now();
```

Just to be consistent with your change